Build failure: hyprlandPlugins.hy3

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

`nix-shell -p hyprlandPlugins.hy3`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/323115626

### Relevant log output

```console
[ 12%] Building CXX object CMakeFiles/hy3.dir/src/main.cpp.o
[ 25%] Building CXX object CMakeFiles/hy3.dir/src/dispatchers.cpp.o
In file included from /build/source/src/globals.hpp:9,
                 from /build/source/src/main.cpp:8:
/build/source/src/Hy3Layout.hpp:15:10: fatal error: hyprland/src/layout/IHyprLayout.hpp: No such file or directory
   15 | #include <hyprland/src/layout/IHyprLayout.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /build/source/src/globals.hpp:9,
                 from /build/source/src/dispatchers.cpp:9:
/build/source/src/Hy3Layout.hpp:15:10: fatal error: hyprland/src/layout/IHyprLayout.hpp: No such file or directory
   15 | #include <hyprland/src/layout/IHyprLayout.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hy3.dir/build.make:79: CMakeFiles/hy3.dir/src/main.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/hy3.dir/build.make:93: CMakeFiles/hy3.dir/src/dispatchers.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hy3.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.64, NixOS, 26.05 (Yarara), 26.05.20260306.aca4d95`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.3`
 - nixpkgs: `/nix/store/3x925rhb3gjikmqkyqigjcdfqm1hsn6a-source`

### Notify maintainers

@aacebedo @johnrtitor

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hy3.nix
+++ b/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hy3.nix
@@ -8,13 +8,13 @@
 }:
 mkHyprlandPlugin (finalAttrs: {
   pluginName = "hy3";
-  version = "0.53.0.1";
+  version = "0.54.2";
 
   src = fetchFromGitHub {
     owner = "outfoxxed";
     repo = "hy3";
     tag = "hl${finalAttrs.version}";
-    hash = "sha256-0gUgUk8XFtu8HWZySFgb4bzvn+MHK9ZEZ1trVU8YfPw=";
+    hash = "sha256-ceT6njnsCLefWhn5C6FYdp68Te9UfcQLAaYIHGjt8bk=";
   };
 
   nativeBuildInputs = [ cmake ];

